### DISCUSSION TITLE
EF Core 6 VS linq2db benchmarks

### DISCUSSION THREAD

**Discussion by vabka** [linq2db/linq2db#3393] [General]
Previously there was benchmarks with different ORMs. 
Where they gone?

If linq2db still faster than EF Core - it would be nice additional argument to choose linq2db over EF

**Comment by MaceWindu** [comment]
You can check https://github.com/FransBouma/RawDataAccessBencher benchmark results.
Unfortunatelly, they currently contain results for v3.4.0 release (we made a lot of perf. improvements in 3.5.x), so it would be nice if somebody send update PR to that repo.

**Reply by vabka** [reply]
Ok. I'll check it.

**Reply by j-dc** [reply]
changed to 3.6.0 d.d. 13/1/2022
https://github.com/FransBouma/RawDataAccessBencher/commit/ce42e4e276cc9aae3a649e09a0acdc4f71896e44#diff-29e19c0c1854c49f924390a09d03dafc06e3dcd457aa1823e0f165c975ed11c3
